enhancements/vdpa/vdpa-introduction.md

```diff
@@ -9,78 +9,45 @@ approvers:
 api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
   - TBD
 creation-date: 2022-07-13
-last-updated: 2022-07-13
-tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
-  - TBD
-see-also:
-  - TBD
-replaces:
-  - TBD
-superseded-by:
-  - TBD
+last-updated: 2022-07-20
+status: provisional
 ---
 
-# VDPA / Openshift integration
-
-Virtual data path acceleration (vDPA) in essence is an approach to standardize the NIC SRIOV data plane using the virtio ring layout and placing a single standard virtio driver in the guest. It’s decoupled from any vendor implementation while adding a generic control plane and SW infrastructure to support it. Given that it’s an abstraction layer on top of SRIOV (Single Root I/O Virtualization) it is also future proof to support emerging technologies such as scalable IOV.
 
-Virtio (see [spec](https://docs.oasis-open.org/virtio/virtio/v1.1/csprd01/virtio-v1.1-csprd01.html)) is used for the data plane while vDPA is used to simplify the control plane thus this solution in general is called virtio/VDPA.
+# Virtio/vDPA with OVN HW offloading
 
-Virtio/vDPA can provide acceleration capabilities replacing SRIOV both for VMs and containers. In this proposal we focus on adding virtio/vDPA capabilities to Openshift SDN as dev-preview for container workloads focusing on Nvidia connectx-6 DX NICs. In the future support for Nvidia Bluefield2/Bluefield3 is planned to be provided as well as support for VMs in openshift (kubevirt and kata containers).
-
-Virtio/vDPA can be consumed in two different ways by the user:
-- virtio-vdpa for container workloads (our focus for this enhancement is the virtio-vdpa solution)
-- vhost-vdpa for DPDK applications, kata containers etc (any VM based app or user space workload). This solution will be addressed in a future enhancement.
+The purpose of this enhancement is to provide a proposal for integrating Virtio/vDPA in OCP for container workloads.
 
 
 ## Summary
+See [VDPA support in OCP (Overview)](https://github.com/openshift/enhancements/blob/master/enhancements/vdpa/overview.md).
+
+Main aspects of this enhancement:
+- primary interface in the pod is configured as a Virtio/vDPA device
+- NIC configured in switchdev mode
+- OVS HW offloading enabled
+- Supported HW: NVIDIA CTX-6 DX
```

Comment:
I think it is better to specify this like so: ConnectX-6 Dx . This is more consistent with NVIDIA documentation.